specification/cosmos-db/data-plane/Microsoft.TablesStorage/preview/2019-02-02/table.json

```diff
@@ -1,7 +1,7 @@
 {
   "swagger": "2.0",
   "info": {
-    "title": "Azure Table Storage",
+    "title": "Azure Table",
     "version": "2019-02-02",
```

Comment:
Does this API version work for both Cosmos and Azure Tables? If yes, we can just remove the 2018 file Otherwise, these changes should be reflected there (they are duplicate files other than the reference to "version").

Reply:
I have been using this API version for both - which generally seems to be working.
Even if that's not the case, we can override the API version at the client, which I think is preferable to maintaining two identical specs. We should just remove the 2018 set if there are no other differences.

Reply:
Sure makes sense